### PR TITLE
Phishing workshop en espanol

### PR DESCRIPTION
**Pull request by beagandica**
Adding spanish translation for the phishing workshop - please ignore branch name.

### CODE REVIEW

File: content/espanol/security-phishing/a-closer-look.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Miremos mas cerca que pasa."
+description: ""
+draft: false
+weight: 3
+---
+
+Bob recibió el siguiente correo electrónico la otra noche.
+
+![Bob's Email](../media/email.png?classes=border,shadow)
```
Comment (ozhang22) [inline content/espanol/security-phishing/a-closer-look.md:10]: Can the alt texts in this file also be translated to Spanish as well?

File: content/espanol/security-phishing/whats-phishy.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+draft: false
+weight: 5
+---
+ 
+ 
+En nuestra última sección aprendimos sobre diferentes tipos de métodos de phishing y cómo reconocerlos. Ahora es el momento de probar tu conocimiento a través de un juego llamado  `¿Qué es phishy?`?
+
+¿Qué imágenes de la lista que se da a continuación crees que son phishy?
+
+![Correo electrónico de Spotify](../media/Spotify_email.jpg?classes=border,shadow)
```
Comment (ozhang22) [inline content/espanol/security-phishing/whats-phishy.md:13]: We will probably need to translate the Phishing image examples into Spanish as well

File: content/espanol/security-phishing/answer-key.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Answer Key - Security"
+description: "Easy reference answer key for the security curriculum"
```
Comment (ozhang22) [inline content/espanol/security-phishing/answer-key.md:3]: Although this is hidden, we should translate these too

File: content/espanol/security-phishing/bob-goes-fishing.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+description: ""
+draft: false
+weight: 1
+---
+
+Conoce a nuestro amigo **Bob**. **Bob** es un caballero agradable y alegre. Sus aficiones son viajar y pescar. Una buena noche, **Bob** recibió un correo electrónico diciendo que ganó un viaje gratis a las "Bahamas". **Bob** se emocionó. Inmediatamente hizo clic en el enlace para ingresar sus datos personales y esperó a que llegara su itinerario.
+
+**Bob** decidió ir de compras para su viaje. Compró ropa nueva y una caña de pescar nueva.
+
+![Bob is preparing for his fishing trip to the Bahamas](../media/nuvi_phish_reduced.png?height=250px)
```
Comment (ozhang22) [inline content/espanol/security-phishing/bob-goes-fishing.md:12]: Can the alt texts in this file also be translated to Spanish as well?

**Review by ozhang22 — APPROVED** [review]

File: content/espanol/security-phishing/answer-key.md (hunk 3 above)
Comment (beagandica) [reply content/espanol/security-phishing/answer-key.md:3]: fixed

File: content/espanol/security-phishing/a-closer-look.md (hunk 1 above)
Comment (ozhang22) [inline content/espanol/security-phishing/a-closer-look.md:10]:
```suggestion
![El correo electrónico de Bob](../media/email.png?classes=border,shadow)
```

File: content/espanol/security-phishing/a-closer-look.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Que crees que esta sospechoso en este correo? Analicemos:
+![Email address](../media/from_address.JPG?classes=border,shadow)
```
Comment (ozhang22) [inline content/espanol/security-phishing/a-closer-look.md:13]:
```suggestion
![dirección de correo electrónico](../media/from_address.JPG?classes=border,shadow)
```

File: content/espanol/security-phishing/a-closer-look.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+
+<div style="text-align: center;">
+<p style="color:red">
+<em>La dirección de correo electrónico del remitente es sospechosa y tu no conoces personalmente a este remitente de correo electrónico.</em></p>
+</div>
+
+![Email Subject](../media/subject.JPG?classes=border,shadow)
```
Comment (ozhang22) [inline content/espanol/security-phishing/a-closer-look.md:21]:
```suggestion
![Asunto del correo electrónico](../media/subject.JPG?classes=border,shadow)
```

File: content/espanol/security-phishing/a-closer-look.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+<div style="text-align: center;">
+<p style="color:red">
+<em>La oferta lucrativa en la línea de asunto sugiere que es demasiado buena para ser verdad.</em></p>
+</div>
+
+![Email Date](../media/date.JPG?classes=border,shadow)
```
Comment (ozhang22) [inline content/espanol/security-phishing/a-closer-look.md:28]:
```suggestion
![Fecha del correo electrónico](../media/date.JPG?classes=border,shadow)
```

File: content/espanol/security-phishing/a-closer-look.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+<div style="text-align: center;">
+<p style="color:red">
+<em>Recibió un correo electrónico fuera de horario comercial.</em></p>
+</div>
+
+![Urgency in language](../media/urgency.JPG?classes=border,shadow)
```
Comment (ozhang22) [inline content/espanol/security-phishing/a-closer-look.md:35]:
```suggestion
![La urgencia en el lenguaje](../media/urgency.JPG?classes=border,shadow)
```

File: content/espanol/security-phishing/a-closer-look.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+<div style="text-align: center;">
+<p style="color:red">
+<em>La urgencia del mensaje.</em></p>
+</div>
+
+![URLs in Email](../media/url.JPG?classes=border,shadow)
```
Comment (ozhang22) [inline content/espanol/security-phishing/a-closer-look.md:42]:
```suggestion
![Las URLs en el correo electrónico](../media/url.JPG?classes=border,shadow)
```

File: content/espanol/security-phishing/bob-goes-fishing.md (hunk 4 above)
Comment (ozhang22) [inline content/espanol/security-phishing/bob-goes-fishing.md:12]:
```suggestion
![Bob se prepara para su viaje de pesca a las Bahamas](../media/nuvi_phish_reduced.png?height=250px)
```

File: content/espanol/security-phishing/bob-goes-fishing.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+Sin embargo, pasaron 24 horas después, y no recibió ninguna confirmación de su viaje. Pasó otro día y todavía nada. Al día siguiente, decidió ir a encontrarse con su buena amiga **Alice**.
+![Secret Agent Alice](../media/Alice_reduced.png?height=250px)
```
Comment (ozhang22) [inline content/espanol/security-phishing/bob-goes-fishing.md:15]:
```suggestion
![Agente secreto Alice](../media/Alice_reduced.png?height=250px)
```

**Review by ozhang22 — APPROVED** [review]
Looks good.

**Pull request merged**